src/alerts/hipchat.coffee

```diff
@@ -11,26 +13,31 @@ module.exports = class HipChatAlert extends Alert
         room_data =
           name: @config.room
         @api.create_room room_data, (err, response) ->
-          if err
-            throw new Error "[Hipchat] Couldn't create room: #{err}"
+          throw new Error "[Hipchat] Couldn't create room: #{err}" if err
 
   sendToHipChat: (message) ->
     # color of the message defaults to yellow if it's falsey in the config
-    options =
-      message: message
+    options = {
+      message
       color: @config.color or 'yellow'
-    @api.notify @config.room, options, (err, response) ->
-      if err
-        throw new Error "[Hipchat] Couldn't send message to room: #{err}"
+    }
+    @api.notify @config.room, options, (err) ->
+      throw new Error "[Hipchat] Couldn't send message to room: #{err}" if err
 
   sendEvent: (event) ->
-    @sendToHipChat [
+    eventString = @renderEvent event
```

Comment:
This pattern of string assignment could be DRY'd up a bit by moving the default format into the Alerts class and doing a:

```coffeescript
eventString = @renderEvent(event) or @defaultEvent(event)
```

Same with the metrics string assignment

```coffeescript
eventString = @renderEvent(event) or @defaultMetricsEvent(event)
```